CHANGELOG.md

```diff
@@ -5,6 +5,19 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/)
 
+## 20240709
+- al2 ami version: 20240709
+- source al2 ami: amzn2-ami-minimal-hvm-2.0.20240620.0-x86_64-ebs
```

Comment:
do we need to include the list of all the source amis? I don't think they've changed since the last release

Reply:
+1